MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include LICENSE
+
+recursive-include test *.py
+
+exclude tox.ini
```

Comment:
It seems strange to exclude `tox.ini` which has our test command yet include the `test` folder. What's the reasoning behind that?